zerver/openapi/zulip.yaml

```diff
@@ -13486,11 +13490,15 @@ paths:
                           for use in settings UI showing the user's avatar.
                       avatar_url:
                         type: string
+                        nullable: true
                         description: |
                           Present if `realm_user` is present in `fetch_event_types`.
 
                           The URL of the avatar for the current user at 100x100
                           resolution. See also `avatar_url_medium`.
+
+                          **Changes**: In Zulip 7.0 (feature level 177)
+                          this will be `null` when client can compute the URL.
```

Comment:
This is also incorrect. The `avatar_url` field for the user themselves is always passed assuming `client_gravatar=False`.